data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/metrics/Metrics.java

```diff
@@ -72,13 +71,23 @@
    */
   public static final String EVENT_DISPATCH_LATENCY = "event_dispatch_latencies";
 
+  /**
+   * @link https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/
+   * @see Metrics#eventDispatchLatency(io.micrometer.core.instrument.Tags)
+   */
+  public static final String EVENT_PROCESSING_LATENCY = "event_processing_latencies";
+
   /**
    * @link https://knative.dev/docs/eventing/observability/metrics/eventing-metrics/
    */
   public static class Tags {
     public static final String RESPONSE_CODE = "response_code";
     public static final String RESPONSE_CODE_CLASS = "response_code_class";
     public static final String EVENT_TYPE = "event_type";
+
+    public static final String RESOURCE_NAME = "name";
+    public static final String RESOURCE_NAMESPACE = "namespace_name";
+    public static final String CONSUMER_NAME = "consumer_name";
```

Comment:
maybe in a follow up PR we can make this dynamic? like an env var sounds like a good idea for sure.

Reply:
Sure, one downside is that we need some form of templating for dashboards (we probably need that anyway, but worth mentioning)